intro_to_python/intro_to_python.md

```diff
@@ -123,149 +115,184 @@ So why should you consider Python for your data analysis? There are a few reason
+
+You can even see what this module's notebook looks like in Google Colab! Once you've worked through the introduction above:
+
+1. Click "File" in the upper left-hand corner and then "Open notebook" in the drop-down menu:
+
+   ![To open a notebook in Google Colab, click the File drop down menu in the top left, and select the second option: Open notebook](media/google_colab_open_nb.png)
 
-### Knowledge Check 4
+ 2. In the window that appears, one of the options at the top is **GitHub**. Click that, and you'll see where you can type in a GitHub URL.
+ 3. In that space, copy and paste the repository where these modules live: [https://github.com/arcus/education_modules](https://github.com/arcus/education_modules).
+ 4. Click the magnifying glass to the far right to search. You should see a list of the Python notebooks in that repository.
+ 5. When you find the one you want, the button on the far right (an upward-slanting arrow in a box) will open the notebook file as a Colab notebook in another tab.
+
+   <div style="display:none">@gifPreload</div>
+
+   <figure>
+     <img src="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.png?raw=true" height="500" width="800" alt="Finding a GitHub repository in Google Colab following the above instructions." data-alt="https://github.com/arcus/education_modules/blob/update-intro-to-python/intro_to_python/media/github_repo_colab.gif?raw=true">
+
+   <figcaption>Click on the image to play the demo.</figcaption>
+   </figure>
+
+### Quiz: Accessing Python on Your Computer
 
 How can you access Python to begin practicing on your own?
 
 [( )] By downloading the latest version at https://www.python.org/downloads/
 [( )] By downloading the latest Anaconda distribution
 [( )] By using cloud-based tools, such as Google Colab
 [(X)] All of the above
-***********************************************************************
+***
 
 <div class = "answer">
 
 You can download Python directly to your computer, use Anaconda to download Python as well as many other data science tools, or use cloud-based notebooks like Google Colab. The choice is yours!
 
 </div>
-***********************************************************************
-
-### Additional Resources
+***
 
-For a beginner-friendly walk-through of writing Python scripts, [Hello World! Your Very First Computer Program](https://youtu.be/cV-QwT4fV7M) delivered by Jeff Pennington is an excellent introduction if you've got an hour to spend.
+## Additional Resources
 
```

Comment:
add link to other python module that was split out from intro to python? (I'm not sure what the new name is, otherwise I'd add it myself as a suggestion! :) )